batch/oak_t.json

```diff
@@ -0,0 +1,14 @@
+{
+    "title": "OAK-T",
+    "description": "OAK-T",
+    "image": "images/oak_d.jpg",
+    "options": { "bootloader": "poe", "imu": true, "usb3": false, "eth": false },
+    "variants": [
+        {
+            "title": "OAK-T (MD2084 R3M2E2)",
```

Comment:
Lets mark this here as well and in EEPROM filename, etc...
CC: @Luxonis-David @ahrastnik  that this nomenclature has to be known by KT as well

Reply:
And everywhere else